### DISCUSSION TITLE
How Ice is working ?

### DISCUSSION THREAD

**Discussion by Kitround** [jordanbaird/Ice#74] [General]
Hello there,

A simple question, how the sections can be arranged in Ice ? I just can't find the solution, or do i really missing something in the options ?

Thank you

**Comment by Kitround** [comment]
Ok i've just found how ! You need to press the command key and drag the icons, maybe it needs a small tutorial on the app page ?

**Reply by septemous** [reply]
Thank you!  Had the same question!

**Comment by 9mm** [comment]
Also had some question. with the huge influx of ppl from bartender probably good to put it as i suspect theyre expecting to re-arrange from inside the app

**Comment by Kitround** [comment]
Well i feel a bit less dumb now, i'm not the only one.

**Comment by jordanbaird** [comment]
Thanks for bringing this up. There's currently a pull request open that adds an explanation on how to do this, along with a video example. It needs a couple changes, but it should be merged in soon. I also plan on adding an onboarding panel with usage examples in the app itself. And the eventual plan is to have items be arrangeable in-app like Bartender as well.

**Reply by Kitround** [reply]
Perfect, and thank you very much for the app.
